server/src/main/java/io/spine/server/integration/TransportFactory.java

```diff
@@ -30,25 +29,26 @@
  * Publish-Subscriber Channel pattern.</a>
  *
  * @author Alex Tymchenko
+ * @author Dmitry Ganzha
  */
 @SPI
 public interface TransportFactory {
 
     /**
-     * Creates a {@link Publisher} for the messages of given class.
+     * Creates a {@link Publisher} for the messages that belong to the kind
```

Comment:
```
Creates a {@link Publisher} for the messages that correspond to the passed channel ID.
```

We also need to have some abstraction, that tells whether the message matches the given channel ID or not. It should probably be implemented in scope of `Channels`. Then we could reference it in this description (e.g.  "correspond" could be a "@linkplain" to that method). We should probably invoke it during the publishing.

Please do that in scope of this PR.  Let's discuss if needed.